recipes/libjpeg-turbo/all/conanfile.py

```diff
@@ -83,14 +83,23 @@ def _configure_cmake(self):
         self._cmake.definitions["WITH_TURBOJPEG"] = self.options.turbojpeg
         self._cmake.definitions["WITH_JAVA"] = self.options.java
         self._cmake.definitions["WITH_12BIT"] = self.options.enable12bit
+        if self.settings.compiler == "Visual Studio":
+            self._cmake.definitions["WITH_CRT_DLL"] = True # avoid replacing /MD by /MT in compiler flags
```

Comment:
should it be applied even for static build?

Reply:
Actually, when enabled, nothing happens. If disabled, it replaces /MD by /MT.

Reply:
okay